trading/trader/trader.py

```diff
@@ -49,22 +50,26 @@ def __init__(self, config, exchange, order_refresh_time=None):
         self.trades_manager = TradesManager(config, self)
 
         self.order_manager = OrdersManager(config, self)
-        self.order_manager_task = None
 
         self.exchange.get_exchange_manager().register_trader(self)
 
         if order_refresh_time is not None:
             self.order_manager.set_order_refresh_time(order_refresh_time)
 
+    async def initialize(self):
```

Comment:
I think we will need an `initialized` method that returns True if initialized else call `initialize`